From 6cd29846d52f3e6411110a1f3eba19a68182864e Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Mon, 18 Jun 2018 13:43:48 +0100
Subject: [PATCH 6/6] manual: thermal-conf.xml.5: fix indentation of XML
 examples
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

The XML examples use a large tab indentation that causes the examples
to wrap on 80 char terminals making it hard to read. Set the indentation
to just 2 characters to make it more readable.  Also fix some .EX example
sections by using correct .EE example end macros.  Fix up some of the
comment blocks too so that wrapping on 80 wide terminals does not occur.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
---
 man/thermal-conf.xml.5 | 553 ++++++++++++++++++++++-------------------
 1 file changed, 295 insertions(+), 258 deletions(-)

diff --git a/man/thermal-conf.xml.5 b/man/thermal-conf.xml.5
index 7b39d22..77d504d 100644
--- a/man/thermal-conf.xml.5
+++ b/man/thermal-conf.xml.5
@@ -21,7 +21,7 @@
 .\"
 .\" Copyright (C) 2012 Intel Corporation. All rights reserved.
 .\"
-.TH thermal-conf.xml "5" "11 Dec 2013"
+.TH thermal-conf.xml "5" "18 Dec 2018"
 
 .SH NAME
 thermal-conf.xml \- Configuration file for thermal daemon
@@ -58,297 +58,334 @@ The thermal sysfs under Linux (/sys/class/thermal) provides a way to represent p
 
 .SH FILE FORMAT
 The configuration file format conforms to XML specifications. A set of tags defined to define platform, sensors, zones, cooling devices and trip points.
+.sp 1
 .EX
 <ThermalConfiguration>
-<Platform>
-	<Name>Example Platform Name</Name>
-	<!--UUID is optional, if present this will be matched. Both product name and UUID can contain wild card "*", which matches any platform.-->
-	<UUID>Example UUID</UUID>
-	configuration file format conforms to XML specifications. A set of tags defined to define platform, sensors, zones,  cooling
-       devices and trip points.
-
-       <ThermalConfiguration>
-
-       <Platform>
-
-
-	<ProductName>Example Product Name</ProductName>
-	<Preference>QUIET|PERFORMANCE</Preference>
-	<!-Quiet mode will only use passive cooling device. "PERFORMANCE" will only select active devices .-->
-	<ThermalSensors>
-		<ThermalSensor>
-			<!-- New Sensor with a type and path -->
-			<Type>example_sensor_1</Type>
-			<Path>/some_path</Path>
-			<AsyncCapable>0</AsyncCapable>
-		</ThermalSensor>
-		<ThermalSensor>
-			<!-- Already present in thermal sysfs, enable this or add/change config For example, here we are indicating that sensor can do async events to avoid polling
-			-->
-			<Type>example_thermal_sysfs_sensor</Type>
-			<!-- If async capable, then we don't need to poll -->
-			<AsyncCapable>1</AsyncCapable>
-		</ThermalSensor>
-	</ThermalSensors>
-	<ThermalZones>
-		<ThermalZone>
-			<Type>Example Zone type</Type>
-			<TripPoints>
-				<TripPoint>
-					<SensorType>example_sensor_1</SensorType>
-					<!-- Temperature at which to take action -->
-					<Temperature> 75000 </Temperature>
-					<!-- max/passive/active If a MAX type is specified, then daemon will use PID control to aggressively throttle to avoid reaching this temp.-->
-					<type>max</type>
-					<!-- SEQUENTIAL | PARALLEL. When a trip point temp is violated, then number of cooling devices can be activated. If control type is SEQUENTIAL then, it will exhaust first cooling device before trying next. -->
-					<ControlType>SEQUENTIAL</ControlType>
-					<CoolingDevice>
-						<index>1</index>
-						<type>example_cooling_device</type>
-						<!-- Influence will be used order cooling devices. First cooling device will be used, which has highest influence. -->
-						<influence> 100 </influence>
-						<!-- Delay in using this cdev, this takes some time too actually cool a zone -->
-						<SamplingPeriod> 12 </SamplingPeriod>
-						<!-- Set a specific state of this cooling device when this trip is violated -->
-						<TargetState> 6 </TargetState>
-						</CoolingDevice>
-				</TripPoint>
-			</TripPoints>
-		</ThermalZone>
-	</ThermalZones>
-	<CoolingDevices>
-		<CoolingDevice>
-			<!-- Cooling device can be specified by a type and optionally a sysfs path. If the type is already present in thermal sysfs, there is no need of a path. Compensation can use min/max and step size to increasing cool the system. Debounce period can be used to force a waiting period for action. -->
-			<Type>example_cooling_device</Type>
-			<MinState>0</MinState>
-			<IncDecStep>10</IncDecStep>
-			<ReadBack> 0 </ReadBack>
-			<MaxState>50</MaxState>
-			<DebouncePeriod>5000</DebouncePeriod>
-			<!-- If there are no PID parameters, compensation increase step wise and exponentially (if single step is not able to change trend). Alternatively a PID parameters can be specified then next step will use PID calculation using provided PID constants. -->
-			<PidControl>
-				<kp>0.001</kp>
-				<kd>0.0001</kd>
-				<ki>0.0001</ki>
-			</PidControl>
-			<!-- Write some prefix attached to state value, like below the prefix is "level ". It will preserve spaces as entered when writing to sysfs -->
-			<WritePrefix>level </WritePrefix>
-		</CoolingDevice>
-	</CoolingDevices>
-</Platform>
-.EX
+  <Platform>
+    <Name>Example Platform Name</Name>
+    <!-- UUID is optional, if present this will be matched. Both product
+         name and UUID can contain wild card "*", which matches any
+         platform. -->
+    <UUID>Example UUID</UUID>
+    <!-- configuration file format conforms to XML specifications. A
+         set of tags defined to define platform, sensors, zones, cooling
+         devices and trip points. -->
+    <ProductName>Example Product Name</ProductName>
+    <Preference>QUIET|PERFORMANCE</Preference>
+    <!-- Quiet mode will only use passive cooling device. "PERFORMANCE"
+         will only select active devices. -->
+    <ThermalSensors>
+      <ThermalSensor>
+        <!-- New Sensor with a type and path -->
+        <Type>example_sensor_1</Type>
+        <Path>/some_path</Path>
+        <AsyncCapable>0</AsyncCapable>
+      </ThermalSensor>
+      <ThermalSensor>
+        <!-- Already present in thermal sysfs, enable this or
+             add/change config For example, here we are indicating
+             that sensor can do async events to avoid polling. -->
+        <Type>example_thermal_sysfs_sensor</Type>
+        <!-- If async capable, then we don't need to poll. -->
+        <AsyncCapable>1</AsyncCapable>
+      </ThermalSensor>
+    </ThermalSensors>
+    <ThermalZones>
+      <ThermalZone>
+        <Type>Example Zone type</Type>
+        <TripPoints>
+          <TripPoint>
+            <SensorType>example_sensor_1</SensorType>
+            <!-- Temperature at which to take action. -->
+            <Temperature> 75000 </Temperature>
+            <!-- max/passive/active If a MAX type is specified, then
+                 daemon will use PID control to aggressively throttle
+                 to avoid reaching this temp. -->
+            <type>max</type>
+            <!-- SEQUENTIAL | PARALLEL. When a trip point temp is
+                 violated, then number of cooling devices can be
+                 activated. If control type is SEQUENTIAL then, it
+                 will exhaust first cooling device
+                 before trying next. -->
+            <ControlType>SEQUENTIAL</ControlType>
+            <CoolingDevice>
+              <index>1</index>
+              <type>example_cooling_device</type>
+              <!-- Influence will be used order cooling devices. First
+                   cooling device will be used, which has highest
+                   influence. -->
+              <influence> 100 </influence>
+              <!-- Delay in using this cdev, this takes some time too
+                   actually cool a zone. -->
+              <SamplingPeriod> 12 </SamplingPeriod>
+              <!-- Set a specific state of this cooling device when this
+                   trip is violated. -->
+              <TargetState> 6 </TargetState>
+            </CoolingDevice>
+          </TripPoint>
+        </TripPoints>
+      </ThermalZone>
+    </ThermalZones>
+    <CoolingDevices>
+      <CoolingDevice>
+        <!-- Cooling device can be specified by a type and optionally
+             a sysfs path. If the type is already present in thermal
+             sysfs, there is no need of a path. Compensation can use
+             min/max and step size to increasing cool the system.
+             Debounce period can be used to force a waiting period
+             for action. -->
+        <Type>example_cooling_device</Type>
+        <MinState>0</MinState>
+        <IncDecStep>10</IncDecStep>
+        <ReadBack> 0 </ReadBack>
+        <MaxState>50</MaxState>
+        <DebouncePeriod>5000</DebouncePeriod>
+        <!-- If there are no PID parameters, compensation increase step
+             wise and exponentially (if single step is not able to
+             change trend).
+             Alternatively a PID parameters can be specified then next
+             step will use PID calculation using provided PID
+             constants. -->
+        <PidControl>
+          <kp>0.001</kp>
+          <kd>0.0001</kd>
+          <ki>0.0001</ki>
+        </PidControl>
+        <!-- Write some prefix attached to state value, like below the
+             prefix is "level ". It will preserve spaces as entered
+             when writing to sysfs. -->
+        <WritePrefix>level </WritePrefix>
+      </CoolingDevice>
+    </CoolingDevices>
+  </Platform>
 </ThermalConfiguration>
+.EE
 .SH EXAMPLE CONFIGURATIONS
 .PP
-Example 1: This is a very simple configuration, to change the passive limit on the CPU. Instead of default, this new temperature 86C in the configuration is used. This will start cooling, once the temperature reaches 86C.
+.B Example 1:
+This is a very simple configuration, to change the passive limit on the CPU. Instead of default, this new temperature 86C in the configuration is used. This will start cooling, once the temperature reaches 86C.
+.sp 1
 .EX
 <?xml version="1.0"?>
 <ThermalConfiguration>
-        <Platform>
-                <Name>Overide CPU default passive</Name>
-                <ProductName>*</ProductName>
-                <Preference>QUIET</Preference>
-                <ThermalZones>
-                        <ThermalZone>
-                                <Type>cpu</Type>
-                                <TripPoints>
-                                                <TripPoint>
-                                                        <Temperature>86000</Temperature>
-                                                        <type>passive</type>
-                                                </TripPoint>
-                                </TripPoints>
-                        </ThermalZone>
-                </ThermalZones>
-        </Platform>
+  <Platform>
+    <Name>Overide CPU default passive</Name>
+    <ProductName>*</ProductName>
+    <Preference>QUIET</Preference>
+    <ThermalZones>
+      <ThermalZone>
+        <Type>cpu</Type>
+        <TripPoints>
+          <TripPoint>
+            <Temperature>86000</Temperature>
+            <type>passive</type>
+          </TripPoint>
+        </TripPoints>
+      </ThermalZone>
+    </ThermalZones>
+  </Platform>
 </ThermalConfiguration>
-.EX
+.EE
 .PP
-Example 2: In this configuration, we are controlling backlight when some sensor "SEN2" reaches 60C. Here "LCD" is a standard cooling device, which uses Linux backlight sysfs interface. "LCD_Zone" is a valid thermal zone in Linux thermal sysfs on the test platform, hence we don't need to provide path for sysfs for "LCD_Zone". The Linux thermal sysfs is already parsed and loaded by thermald program.
+.B Example 2:
+In this configuration, we are controlling backlight when some sensor "SEN2" reaches 60C. Here "LCD" is a standard cooling device, which uses Linux backlight sysfs interface. "LCD_Zone" is a valid thermal zone in Linux thermal sysfs on the test platform, hence we don't need to provide path for sysfs for "LCD_Zone". The Linux thermal sysfs is already parsed and loaded by thermald program.
+.sp 1
 .EX
 <?xml version="1.0"?>
 <ThermalConfiguration>
-	<Platform>
-		<Name>Change Backlight</Name>
-		<ProductName>*</ProductName>
-		<Preference>QUIET</Preference>
-		<ThermalZones>
-			<ThermalZone>
-				<Type>LCD_Zone</Type>
-				<TripPoints>
-						<TripPoint>
-							<SensorType>SEN2</SensorType>
-							<Temperature>60000</Temperature>
-							<type>passive</type>
-						<CoolingDevice>
-							<Type>LCD</Type>
-						</CoolingDevice>
-						</TripPoint>
-				</TripPoints>
-			</ThermalZone>
-		</ThermalZones>
-	</Platform>
+  <Platform>
+    <Name>Change Backlight</Name>
+    <ProductName>*</ProductName>
+    <Preference>QUIET</Preference>
+    <ThermalZones>
+      <ThermalZone>
+        <Type>LCD_Zone</Type>
+        <TripPoints>
+          <TripPoint>
+            <SensorType>SEN2</SensorType>
+            <Temperature>60000</Temperature>
+            <type>passive</type>
+            <CoolingDevice>
+              <Type>LCD</Type>
+            </CoolingDevice>
+          </TripPoint>
+        </TripPoints>
+      </ThermalZone>
+    </ThermalZones>
+  </Platform>
 </ThermalConfiguration>
 .EE
-Example 3: In this example Lenovo Thinkpad X220 and fan speed is controlled. Here a cooling device "_Fan", can be controlled via sysfs
+.PP
+.B Example 3:
+In this example Lenovo Thinkpad X220 and fan speed is controlled. Here a cooling device "_Fan", can be controlled via sysfs
 /sys/devices/platform/thinkpad_hwmon/pwm1. When the x86_pkg_temp reaches 45C, Fan is started with increasing speeds, if the temperature can't be controlled at 45C.
+.sp 1
 .EX
 <?xml version="1.0"?>
 <ThermalConfiguration>
-<Platform>
-<Name>Lenovo ThinkPad X220</Name>
-<ProductName>*</ProductName>
-<Preference>QUIET</Preference>
-<ThermalZones>
-                <ThermalZone>
-                                <Type>x86_pkg_temp</Type>
-                                <TripPoints>
-                                                <TripPoint>
-                                                                <SensorType>x86_pkg_temp</SensorType>
-                                                                <Temperature>45000</Temperature>
-                                                                <type>passive</type>
-                                                                <ControlType>SEQUENTIAL</ControlType>
-                                                                <CoolingDevice>
-                                                                                <index>1</index>
-                                                                                <type>_Fan</type>
-                                                                                <influence> 100 </influence>
-                                                                                <SamplingPeriod> 12 </SamplingPeriod>
-                                                                </CoolingDevice>
-                                                </TripPoint>
-                                </TripPoints>
-                </ThermalZone>
-</ThermalZones>
-<CoolingDevices>
-                <CoolingDevice>
-                                <Type>_Fan</Type>
-                                <Path>/sys/devices/platform/thinkpad_hwmon/pwm1</Path>
-                                <MinState>0</MinState>
-                                <IncDecStep>30</IncDecStep>
-                                <ReadBack> 0 </ReadBack>
-                                <MaxState>255</MaxState>
-                                <DebouncePeriod>5</DebouncePeriod>
-                </CoolingDevice>
-</CoolingDevices>
-</Platform>
+  <Platform>
+    <Name>Lenovo ThinkPad X220</Name>
+    <ProductName>*</ProductName>
+    <Preference>QUIET</Preference>
+    <ThermalZones>
+      <ThermalZone>
+        <Type>x86_pkg_temp</Type>
+        <TripPoints>
+          <TripPoint>
+            <SensorType>x86_pkg_temp</SensorType>
+            <Temperature>45000</Temperature>
+            <type>passive</type>
+            <ControlType>SEQUENTIAL</ControlType>
+            <CoolingDevice>
+              <index>1</index>
+              <type>_Fan</type>
+              <influence> 100 </influence>
+              <SamplingPeriod> 12 </SamplingPeriod>
+            </CoolingDevice>
+          </TripPoint>
+        </TripPoints>
+      </ThermalZone>
+    </ThermalZones>
+    <CoolingDevices>
+      <CoolingDevice>
+        <Type>_Fan</Type>
+        <Path>/sys/devices/platform/thinkpad_hwmon/pwm1</Path>
+        <MinState>0</MinState>
+        <IncDecStep>30</IncDecStep>
+        <ReadBack> 0 </ReadBack>
+        <MaxState>255</MaxState>
+        <DebouncePeriod>5</DebouncePeriod>
+      </CoolingDevice>
+    </CoolingDevices>
+  </Platform>
 </ThermalConfiguration>
-.EX
-Example 4: The following example shows how PID can be used. Here once temperature exceeds 80C, compensation is calculated using PID using 80C as set point of PID. The compensation depends on error from the set point. Here the default built in processor cooling device is used with min state as 0 and max state as 10.
+.EE
+.PP
+.B Example 4:
+The following example shows how PID can be used. Here once temperature exceeds 80C, compensation is calculated using PID using 80C as set point of PID. The compensation depends on error from the set point. Here the default built in processor cooling device is used with min state as 0 and max state as 10.
+.sp 1
 .EX
 <?xml version="1.0"?>
 <ThermalConfiguration>
-<Platform>
-<Name>Use PID param </Name>
-<ProductName>*</ProductName>
-<Preference>QUIET</Preference>
-<ThermalZones>
-                <ThermalZone>
-                                <Type>x86_pkg_temp</Type>
-                                <TripPoints>
-                                                <TripPoint>
-                                                                <SensorType>x86_pkg_temp</SensorType>
-                                                                <Temperature>80000</Temperature>
-                                                                <type>passive</type>
-                                                                <ControlType>SEQUENTIAL</ControlType>
-                                                                <CoolingDevice>
-									<type>Processor</type>
-                                                                </CoolingDevice>
-                                                </TripPoint>
-                                </TripPoints>
-                </ThermalZone>
-</ThermalZones>
- <CoolingDevices>
-               <CoolingDevice>
-                                <Type>Processor</Type>
-                                <PidControl>
-                                              <kp>0.0002</kp>
-                                              <kd>0</kd>
-                                              <ki>0</ki>
-                                </PidControl>
-              </CoolingDevice>
-</CoolingDevices>
-</Platform>
+  <Platform>
+    <Name>Use PID param </Name>
+    <ProductName>*</ProductName>
+    <Preference>QUIET</Preference>
+    <ThermalZones>
+      <ThermalZone>
+        <Type>x86_pkg_temp</Type>
+        <TripPoints>
+          <TripPoint>
+            <SensorType>x86_pkg_temp</SensorType>
+            <Temperature>80000</Temperature>
+            <type>passive</type>
+            <ControlType>SEQUENTIAL</ControlType>
+            <CoolingDevice>
+              <type>Processor</type>
+            </CoolingDevice>
+          </TripPoint>
+        </TripPoints>
+      </ThermalZone>
+    </ThermalZones>
+    <CoolingDevices>
+      <CoolingDevice>
+        <Type>Processor</Type>
+        <PidControl>
+          <kp>0.0002</kp>
+          <kd>0</kd>
+          <ki>0</ki>
+        </PidControl>
+      </CoolingDevice>
+    </CoolingDevices>
+  </Platform>
 </ThermalConfiguration>
 .EE
-Example 5: The following example shows how to control Fan when the sysfs expects some string prefix. For example instead of just write a number to fan control sysfs, the interface requires "level " in front of the speed index value.
+.PP
+.B Example 5:
+The following example shows how to control Fan when the sysfs expects some string prefix. For example instead of just write a number to fan control sysfs, the interface requires "level " in front of the speed index value.
+.sp 1
 .EX
 <?xml version="1.0"?>
 <ThermalConfiguration>
-	<Platform>
-		<Name>Use Fan control first then CPU throttle </Name>
-		<ProductName>*</ProductName>
-		<Preference>QUIET</Preference>
-		<ThermalZones>
-                       <ThermalZone>
-                                       <Type>x86_pkg_temp</Type>
-                                       <TripPoints>
-                                                       <TripPoint>
-                                                                       <SensorType>x86_pkg_temp</SensorType>
-                                                                       <Temperature>80000</Temperature>
-									<type>passive</type>
-                                                                       <ControlType>SEQUENTIAL</ControlType>
-                                                                       <CoolingDevice>
-                                                    <type>_fan_</type>
-                                                                       </CoolingDevice>
-                                                       </TripPoint>
-                                       </TripPoints>
-                       </ThermalZone>
-                </ThermalZones>
-                <CoolingDevices>
-                      <CoolingDevice>
-                                       <Type>_fan_</Type>
-					<Path>/proc/acpi/ibm/fan</Path>
-					<WritePrefix>level </WritePrefix>
-					<MinState>0</MinState>
-					<MaxState>5</MaxState>
-					<DebouncePeriod>10</DebouncePeriod>
-                     </CoolingDevice>
-                </CoolingDevices>
-       </Platform>
+  <Platform>
+    <Name>Use Fan control first then CPU throttle </Name>
+    <ProductName>*</ProductName>
+    <Preference>QUIET</Preference>
+    <ThermalZones>
+      <ThermalZone>
+        <Type>x86_pkg_temp</Type>
+        <TripPoints>
+          <TripPoint>
+            <SensorType>x86_pkg_temp</SensorType>
+            <Temperature>80000</Temperature>
+            <type>passive</type>
+            <ControlType>SEQUENTIAL</ControlType>
+            <CoolingDevice>
+              <type>_fan_</type>
+            </CoolingDevice>
+          </TripPoint>
+        </TripPoints>
+      </ThermalZone>
+    </ThermalZones>
+    <CoolingDevices>
+      <CoolingDevice>
+        <Type>_fan_</Type>
+        <Path>/proc/acpi/ibm/fan</Path>
+        <WritePrefix>level </WritePrefix>
+        <MinState>0</MinState>
+        <MaxState>5</MaxState>
+        <DebouncePeriod>10</DebouncePeriod>
+      </CoolingDevice>
+    </CoolingDevices>
+  </Platform>
 </ThermalConfiguration>
 .EE
+.PP
 Example 6: Similar to example 5, but write different speeds at
 different temperatures.
+.sp 1
 .EX
 <?xml version="1.0"?>
 <ThermalConfiguration>
-	<Platform>
-		<Name>Use Fan control first then CPU throttle </Name>
-		<ProductName>*</ProductName>
-		<Preference>QUIET</Preference>
-		<ThermalZones>
-                       <ThermalZone>
-                                       <Type>x86_pkg_temp</Type>
-                                       <TripPoints>
-                                                       <TripPoint>
-                                                                       <SensorType>x86_pkg_temp</SensorType>
-                                                                       <Temperature>80000</Temperature>
-									<type>passive</type>
-                                                                       <CoolingDevice>
-                                                    <type>_fan_</type>
-                                                                       <TargetState>1</TargetState>
-                                                                       </CoolingDevice>
-                                                       </TripPoint>
-                                                       <TripPoint>
-                                                                       <SensorType>x86_pkg_temp</SensorType>
-                                                                       <Temperature>85000</Temperature>
-									<type>passive</type>
-                                                                       <CoolingDevice>
-                                                    <type>_fan_</type>
-                                                                       <TargetState>2</TargetState>
-                                                                       </CoolingDevice>
-                                                       </TripPoint>
-                                       </TripPoints>
-                       </ThermalZone>
-                </ThermalZones>
-                <CoolingDevices>
-                      <CoolingDevice>
-                                       <Type>_fan_</Type>
-					<Path>/proc/acpi/ibm/fan</Path>
-					<WritePrefix>level </WritePrefix>
-					<MinState>0</MinState>
-					<MaxState>5</MaxState>
-					<DebouncePeriod>10</DebouncePeriod>
-                     </CoolingDevice>
-               </CoolingDevices>
-       </Platform>
+  <Platform>
+    <Name>Use Fan control first then CPU throttle </Name>
+    <ProductName>*</ProductName>
+    <Preference>QUIET</Preference>
+    <ThermalZones>
+      <ThermalZone>
+        <Type>x86_pkg_temp</Type>
+        <TripPoints>
+          <TripPoint>
+            <SensorType>x86_pkg_temp</SensorType>
+            <Temperature>80000</Temperature>
+            <type>passive</type>
+            <CoolingDevice>
+              <type>_fan_</type>
+              <TargetState>1</TargetState>
+            </CoolingDevice>
+          </TripPoint>
+          <TripPoint>
+            <SensorType>x86_pkg_temp</SensorType>
+            <Temperature>85000</Temperature>
+            <type>passive</type>
+            <CoolingDevice>
+              <type>_fan_</type>
+              <TargetState>2</TargetState>
+            </CoolingDevice>
+          </TripPoint>
+        </TripPoints>
+      </ThermalZone>
+    </ThermalZones>
+    <CoolingDevices>
+      <CoolingDevice>
+        <Type>_fan_</Type>
+        <Path>/proc/acpi/ibm/fan</Path>
+        <WritePrefix>level </WritePrefix>
+        <MinState>0</MinState>
+        <MaxState>5</MaxState>
+        <DebouncePeriod>10</DebouncePeriod>
+      </CoolingDevice>
+    </CoolingDevices>
+  </Platform>
 </ThermalConfiguration>
 .EE
-- 
2.17.0

